docfx_project/articles/bootstrapping-trust/part-05.bootstrapping-trust.md

```diff
@@ -0,0 +1,145 @@
+Bootstrapping Trust
+
+## Part 5 Bootstrapping Trust
+
+If you have read through the entire series up to here, welcome! If you
+have not, please consider reading the whole series:
+
+- [Part 1: Encryption Everywhere](./part-01.encryption-everywhere.md)
+- [Part 2: A Primer On Public-Key Cryptography](./part-02.a-primer-on-public-key-cryptography.md)
+- [Part 3: Certificates](./part-03.certificates.md)
+- [Part 4: Certificate Authorities & Chains Of Trust](./part-04.certificate-authorities-and-chains-of-trust.md)
+- [Part 5: Bootstrapping Trust](./part-05.bootstrapping-trust.md)
+
+### Ziti
+
+In this series of articles are exploring bootstrapping trust and what
+that means. Part of the ZIti software system is to provide a method to
```

Comment:
"Part of the ZIti software system is to provide a method" - maybe just "Ziti provides a method to bootstrap trust via its enrollment process"?  at the very least - capital `I` in `ZIti` needs to be fixed

Reply:
reworded